core/conn/odb/src/odb.c

```diff
@@ -10076,6 +10114,7 @@ static int Oloadbuff(int eid)
         clock_gettime(CLOCK_MONOTONIC, &tsp1);
 #endif
         Or = SQLExecute(thps[tid].Os) ;         /* Execute INSERT (load/copy) or tgt command */
+        SQLLEN tLastRow = -1;           /* this is special solution for China Union Pay, print only first error message for state 22003 */
```

Comment:
Probably best not to put customer names into the code. Consider changing the comment to just "/* print only first error message for state 22003 */"

Reply:
Thank you, I will fix this.